### DISCUSSION TITLE
Are there any known issues with GoToNextError / GoToPreviousError ?

### DISCUSSION THREAD

**Discussion by ozzythecoder** [JetBrains/ideavim#1025] [Q&A] [answered]
Hey friends,

I'm enjoying IdeaVim in Webstorm quite a bit, I'm just having some trouble with the `GoToNextError` and `GoToPreviousError` actions - namely that the action itself doesn't seem to work, even when I type `:action GoToNextError` directly into the command line.

What I've tried:
- `map ge`
- `map <leader>ge`
- `map \e`
- `nmap \e`
- mapping to `<Action>(GoToNextError)`
- mapping to `:action GoToNextError<CR>`
- mapping to default VScode MacOS keymaps `<M-F8>` / `<S-F8>`
- Deactivating all other plugins
- Deleting the default VScode MacOS keymaps

All my other keymaps are doing just fine. I've done some searching and haven't seen anybody else with my specific issue. Am I misunderstanding something about the functionality of the action?

My ideavimrc is below.
```vimscript
# .ideavimrc

set nu
set relativenumber
set scrolloff=5
imap jj <Esc>
let mapleader=" "
map \w :w<CR>

if has('ide')
  " code actions
  map K <Action>(ShowHoverInfo)
  map \a <Action>(ShowIntentionActions)
  map \e <Action>(RenameElement)
  map \r <Action>(Javascript.Linters.EsLint.Fix)
  map \f <Action>(ReformatCode)
  map \s <Action>(SurroundWith)
  map \0 <Action>(Run)
  map \x <Action>(Stop)
  map \t <Action>(ActivateTerminalToolWindow)
  map \q <Action>(HideActiveWindow)
  map \[ <Action>(ActivateCodeiumChatToolWindow)

  map ,e <Action>(GoToNextError)
  map ,E <Action>(GoToPreviousError)

  " navigation
  map <leader>e <Action>(SelectInProjectView)
  map <leader>f <Action>(GotoFile)
  map <leader>b <Action>(Switcher)
  map <leader>x <Action>(CloseContent)
  map <leader>p <Action>(SearchEverywhere)
  map <Tab> <Action>(NextTab)
  map <S-Tab> <Action>(PreviousTab)
  map <leader><Tab> <Action>(NextSplitter)

  nnoremap <leader>si :source ~/.ideavimrc<CR>
endif
```

**Comment by citizenmatt** [comment]
I'm not aware of any issues. I haven't checked WebStorm, but I use a similar mapping in Rider and it works ok. Does the main WebStorm shortcut work to move to the next/previous error?

**Reply by ozzythecoder** [reply]
Hey, yeah the main shortcut does work, as well as choosing "Next Highlighted Error" from the menu or command palette. This will also show the GoToNextError tracking ID too.

I'd love to be able to remap these with ideavim, as Fn+F8 is a nasty reach on a MacBook keyboard and I'd prefer to stick to my leader keys.

**Reply by citizenmatt** [reply, marked as answer]
Please try the mapping with lowercase `t` - `<Action>(GotoNextError)` and `<Action>(GotoPreviousError)`. I didn't think action ID lookup was case sensitive, but I guess it is.

**Reply by ozzythecoder** [reply]
Oh wow, I completely missed that it was `Goto` and not `GoTo`. This solved all of my problems, lol. Thank you!
